.github/workflows/test-and-release.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [18.x, 20.x, 22.x]
+        node-version: [18.x, 20.x, 22.4.1]
```

Comment:
Is there a reason to be that precise?^

Reply:
yes a current npm bug ... else you get https://github.com/ioBroker/ioBroker.matter/actions/runs/10021413829/job/27700078813#step:4:16 ... maybe we can revert when npm fixed that